lib/engine_credo/config.ex

```diff
@@ -35,9 +36,9 @@ defmodule EngineCredo.Config do
       |> Credo.Config.read_or_default(nil, true) # true required for safe loading of `.credo.exs`
       |> include_files_from(engine_config, path)
 
-    source_files = valid_source_files(credo_config)
+    {source_files, invalid_files} = Enum.partition(find_source_files(credo_config), &(&1.valid?))
```

Comment:
💅 Now that we requalified that function I think we can make it do more, like this:

```elixir
{source_files, invalid_files} = find_source_files(credo_config)

  defp find_source_files(config) do
    config
    |> Credo.Sources.find
    |> Enum.filter(&String.ends_with?(&1.filename, [".ex", ".exs"]))
    |> Enum.partition(&(&1.valid?))
  end
```

WDYT?